instrumentation/google.golang.org/grpc/otelgrpc/config.go

```diff
@@ -41,6 +41,8 @@ type config struct {
 	Propagators    propagation.TextMapPropagator
 	TracerProvider trace.TracerProvider
 	MeterProvider  metric.MeterProvider
+	ReceivedEvent  bool
```

Comment:
Could you add a new line before those attributes (and in the other structs as well)? So they are split from the other options.

Reply:
Fixed.